frontend/src/routes/FacilityBooking/ViewAllFacilities/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { useHistory } from 'react-router-dom'
+
+import 'antd/dist/antd.css'
+
+import { PATHS } from '../../Routes'
+import TopNavBar from '../../../components/Mobile/TopNavBar'
+import BottomNavBar from '../../../components/Mobile/BottomNavBar'
+import LoadingSpin from '../../../components/LoadingSpin'
+import FacilitiesList from '../../../components/FacilityBooking/FacilitiesList'
+
+import {
+  getFacilityList,
+  setIsLoading,
+  setSelectedFacility,
+  SetBlockOutIsOpen,
+  sortFacilitiesAlphabetically,
+} from '../../../store/facilityBooking/action'
+import { RootState } from '../../../store/types'
+
+import bookingsIcon from '../../../assets/bookingsIcon.svg'
+import JCRCBlockOutIcon from '../../../assets/JCRCBlockOut.svg'
+import TopNavBarRevamp from '../../../components/TopNavBarRevamp'
+import MyBookingsIcon from '../../../components/FacilityBooking/MyBookingsIcon'
+import styled from 'styled-components'
+
+const MainContainer = styled.div`
+  height: 100%;
+  width: 100%;
+  background-color: #fafaf4;
+`
+
+const TitleText = styled.h2`
+  font-family: Lato;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 22px;
+  margin-top: 0.7rem;
+`
+
+/**
+ * # View All Facilities Page
+ * Path: `/facility/allFacilities`
+ *
+ * ## Page Description
+ * View All Facilities Page is accessable through the Facilities Landing Page. On FacilitiesBooking page, users will be shown
+ * all the facilities (venue) that can be booked for events. Upon selection of facility, user will
+ * be directed to the selected facility page to select the date & timing for booking.
+ *
+ * @remarks
+ */
+export default function ViewAllFacilities() {
```

Comment:
consider writing in es6 arrow functions for new ones.

Reply:
uwu yes sire